### PR TITLE
Added method for getting individual player details and matchups

### PR DESCRIPTION
**Pull request by DMcP89**

### CODE REVIEW

**Comment by DMcP89** [comment]
@spilchen  I believe I have made all the changes you've requested. I've also added another method for get matchups data.

File: yahoo_fantasy_api/yhandler.py [hunk 1]
```diff
@@ -114,3 +114,21 @@ def get_players_raw(self, league_id, start, status, position=None):
             pos_parm = ";position={}".format(position)
         return self.get("league/{}/players;start={};count=25;status={}{}".
                         format(league_id, start, status, pos_parm))
+<<<<<<< HEAD
```
Comment (spilchen) [inline yahoo_fantasy_api/yhandler.py]: Looks like you still have the git conflict markers in this file.

File: yahoo_fantasy_api/league.py [hunk 2]
```diff
@@ -284,3 +294,56 @@ def _date_range_of_played_or_current_week(self, week):
...
+        :return: Player details
+        :rtype: dict
+        """
+        """Retrieve details about a specific player
+        
+        :parm player_name: Name of the player to get details for
+        :type player_name: str
+        :return: Player details
+        :rtype: dict
+        >>> lg.player_details("Antonio Brown")
```
Comment (spilchen) [inline yahoo_fantasy_api/league.py]: I like the extra detail to show what the output looks like.  But part of this docstring is duplicated above.

File: yahoo_fantasy_api/league.py [hunk 3]
```diff
@@ -4,6 +4,7 @@
 from yahoo_fantasy_api import yhandler
 import objectpath
 import datetime
+import pdb
```
Comment (spilchen) [inline yahoo_fantasy_api/league.py]: Please remove this import as I don't see it being used.

**Comment by DMcP89** [comment]
@spilchen  should be all set now. Let me know if there is anything else.

**Pull request merged**